database/factories/RoomFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Type;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Room>
+ */
+class RoomFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'id' => $this->faker->numberBetween(1, 100),
+            'name' => $this->faker->text(255),
+            'rate' => rand(1, 100),
+            'image' => $this->faker->imageUrl,
```

Comment:
Can you make sure the image is 400x400 please? You should be able to do `$this->faker->imageUrl(400, 400)`